server/config/seed.js

```diff
@@ -85,14 +85,12 @@ var seed = function(cb) {
                 console.log('finished populating class years')
             })
 
-    Promise.all([user, project, post, smallgroup, classYear]).then(function(res){
-        cb();
-    });
+    return Promise.all([user, project, post, smallgroup, classYear]);
```

Comment:
no longer disconnecting from the DB? Is this a good thing? not sure how that works